### ISSUE TITLE
Sum repeated ingredients on the overview

### ISSUE THREAD

**Issue by igorsantos07** [TandoorRecipes/recipes#3450]
### Is your feature request related to a problem? Please describe.

Ingredients split between different steps are shown separately on the summary at the top of the recipe (besides the picture).

### Describe the solution you'd like

There should be an option to see all ingredients in a single list, effectively adding up repeated ones - so it's easier in a glance to prepare a shopping list, in case you don't use Tandoor's.

An idea could be to have that as a switch on the right side of the "Ingredients" header.

### Additional context

![image](https://github.com/user-attachments/assets/24b91452-58ec-4126-b09c-e9e541db1760)

**Comment by smilerz** [comment]
Duplicate

**Comment by igorsantos07** [comment]
Indeed, my bad. There are at least two other issues where other people reported the exact same problem:
- #3401 - a month ago
- #595 - 2021, closed even though the reporter said it's not solved
Given this is the third time the problem is reported, and all issues got closed without a valid reason (other than a related feature being implemented), and also given I made a suggestion that could solve the problem without major trouble (e.g. changing per-recipe options), I guess this issue should be reopened, or at least #595 should - it seems to be the original one.
